content/en/blog/_posts/get-started-with-kubernetes-using-python.md

```diff
@@ -0,0 +1,217 @@
+* [Docker Desktop with Kubernetes](https://www.docker.com/products/kubernetes) installed (see end of blog for enabling Kubernetes)
+* [Python 3.7](https://www.python.org/) installed
+* [Git](https://git-scm.com/downloads) installed
+
+## Containerizing an application
+In this section you’ll take some source code, verify it runs locally, and then create a Docker image of the application. The sample application used is a very simple Flask web application; if you want to test it locally, you’ll need Python installed. Otherwise, you can skip to the "Create a Dockerfile" section.
+
+### Get the application code
+
+Use git to clone the repository to your local machine:
+```
+git clone https://github.com/JasonHaley/hello-python.git
+```
+
+Change to the app directory:
+```
+cd hello-python/app
+```
+
+There are only two files in this directory. If you look at the main.py file, you’ll see the application prints out a hello message. You can learn more about Flask on the [Flask website](http://flask.pocoo.org/).
+
+```
+from flask import Flask
+app = Flask(__name__)
+
+@app.route("/")
+def hello():
+    return "Hello from Python!"
+
+if __name__ == "__main__":
+    app.run(host='0.0.0.0')
+```
+
+The requirements.txt file contains the list of packages needed by the main.py and will be used by [pip](https://pip.pypa.io/en/stable/) to install the Flask library.
+
+### Run locally
+Manually run the installer and application using the following commands:
+```
+pip install -r requirements.txt
+python main.py
+```
+
+This will start a development web server hosting your application, which you will be able to see by navigating to http://localhost:5000. Because port 5000 is the default port for the development server, we didn’t need to specify it.
+
+### Create a Dockerfile
+Now that you have verified the source code works, the first step in containerizing the application is to create a Dockerfile.
+
+In the hello-python/app directory, create a file named Dockerfile with the following contents and save it:
+```
+FROM python:3.7
+
+RUN mkdir /app
+WORKDIR /app
+ADD . /app/
+RUN pip install -r requirements.txt
+
+EXPOSE 5000
+CMD ["python", "/app/main.py"]
+```
+
+This file is a set of instructions Docker will use to build the image. For this simple application, Docker is going to:
+
+1. Get the official [Python Base Image](https://hub.docker.com/_/python/) for version 3.7 from Docker Hub.
+2. In the image, create a directory named app.
+3. Set the working directory to that new app directory.
+4. Copy the local directory’s contents to that new folder into the image.
+5. Run the pip installer (just like we did earlier) to pull the requirements into the image.
+6. Configure port 5000 to be open.
+7. Configure the starting command to use when the container starts.
+
+### Create an image
+At your command line or shell, in the hello-python/app directory, build the image with the following command:
+```
+docker build -f Dockerfile -t hello-python:latest .
```

Comment:
Might be worth mentioning in a note here that using the `:latest` tag on an image can be error-prone. See here for more information: https://container-solutions.com/docker-latest-confusion/

Reply:
I added a note about that and pointed to the link, but I'm not sure if it belongs. See what you think.